src/processor/operator/aggregate/aggregate_hash_table.cpp

```diff
@@ -639,6 +639,52 @@ static bool compareEntry(common::ValueVector* vector, uint32_t vectorPos, const
     return result != 0;
 }
 
+template<>
+bool compareEntry<list_entry_t>(
+    common::ValueVector* vector, uint32_t vectorPos, const uint8_t* entry) {
+    auto dataVector = ListVector::getDataVector(vector);
+    auto listToCompare = vector->getValue<list_entry_t>(vectorPos);
+    auto listEntry = reinterpret_cast<const ku_list_t*>(entry);
+    auto entryNullBytes = reinterpret_cast<uint8_t*>(listEntry->overflowPtr);
+    auto entryValues = entryNullBytes + NullBuffer::getNumBytesForNullValues(listEntry->size);
+    auto rowLayoutSize = LogicalTypeUtils::getRowLayoutSize(dataVector->dataType);
+    compare_function_t compareFunc;
+    AggregateHashTable::getCompareEntryWithKeysFunc(dataVector->dataType, compareFunc);
```

Comment:
Let's discuss if we can make wrap compare function as a nested data structure so we don't need to resolve it at compile time.